yt_dlp/extractor/lecturio.py

```diff
@@ -73,6 +73,9 @@ class LecturioIE(LecturioBaseIE):
     }, {
         'url': 'https://www.lecturio.de/jura/oeffentliches-recht-staatsexamen.vortrag',
         'only_matching': True,
+    }, {
+        'url': 'https://www.lecturio.de/jura/oeffentliches-recht-at-1-staatsexamen/oeffentliches-recht-staatsexamen.vortrag',
```

Comment:
test was added but seems not effective? Running `python3 test/test_download.py TestDownload.test_Lecturio_all ` won't have any error even before patch was applied

Reply:
`only_matching` cases are tested by `python3 test/test_all_urls.py TestAllURLsMatching.test_no_duplicates`

Reply:
Ohh